share/qtcreator/sfdk/modules/20-building-mb2/doc/module.adoc

```diff
@@ -3,18 +3,19 @@
 Start by choosing the build target. List the available build tools.
 
     $ sfdk tools list
-    %{OsVariant:NoSpace}-3.0.0.8
-    └─%{OsVariant:NoSpace}-3.0.0.8-i486
-    %{OsVariant:NoSpace}-3.0.2.8
-    ├─%{OsVariant:NoSpace}-3.0.2.8-armv7hl
-    └─%{OsVariant:NoSpace}-3.0.2.8-i486
+    %{OsVariant:NoSpace}-4.1.0.24
+    └─%{OsVariant:NoSpace}-4.1.0.24-i486
+    %{OsVariant:NoSpace}-4.2.0.21
+    ├─%{OsVariant:NoSpace}-4.2.0.21-aarch64
+    ├─%{OsVariant:NoSpace}-4.2.0.21-armv7hl
+    └─%{OsVariant:NoSpace}-4.2.0.21-i486
```

Comment:
Why not go straight to 4.3.0.12?

Reply:
I use older example version numbers to make it more obvious that these are static examples hanging behind the reality.

Reply:
makes sense